package.json

```diff
@@ -26,6 +26,7 @@
     "@babel/plugin-transform-runtime": "^7.0.0",
     "@babel/preset-env": "^7.12.11",
     "@babel/runtime": "^7.0.0",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
```

Comment:
I think we technically don't need this anymore (`npm rm "@fortawesome/free-solid-svg-icons"` should do the trick)